.cirrus.yml

```diff
@@ -756,6 +756,22 @@ rootless_system_test_task:
     main_script: *main
     always: *logs_artifacts
 
+minikube_test_task:
+    name: *std_name_fmt
+    alias: minikube_test
+    # Docs: ./contrib/cirrus/CIModes.md
+    only_if: *not_tag_build_docs_multiarch
+    depends_on:
+        - build
+        - rootless_system_test
+    gce_instance: *standardvm
+    env:
+        <<: *stdenvars
+        TEST_FLAVOR: minikube
```

Comment:
>can minikube be run rootless? root privilege are not required for it when using podman

@umohnani8 I interpret your question as "how can we make this CI test run rootless". If that interpretation is correct, I believe the answer is, add `PRIV_NAME: rootless` immediately after this TEST_FLAVOR one, at the same indentation. (If you're asking about minikube itself, I can't help).

Reply:
Correct.  Though reading through the issue a bit, it seems to center around having a FQDN hostname set.  That should **not** be the case here. The CI hostnames should all be `cirrus-task-<blahblahblah>` or `localhost`.  In any case, I'm fine either way (root or rootless).  Though rootless brings significant additional setup complexity to the environment.  That may be worthy of avoiding if the simple/suggested `--driver=none` could work?

Reply:
yup, that is exactly what I wanted and it works, thanks!